### ISSUE TITLE
Not raising exception as encrypting empty string with mock_kms

### ISSUE THREAD

**Issue by yuecen** [spulec/moto#2426]
Hi,

I used `mock_kms` to mock out a KMS service during unit test. Everything works fine until I deployed my code to AWS and run it. It seems that an empty string can be encrypted with moto_kms, but fails with the real KMS service.

- The moto version: 1.3.13
- Python: 3.7.4

```python
>>> from moto import mock_kms                                                                                                                  
>>> mock_kms = mock_kms()
>>> mock_kms.start()
>>> import boto3
>>> client_kms = boto3.client('kms', region_name='ap-northeast-1')                                                                             
>>> client_kms.encrypt(KeyId='kms-id', Plaintext='')                                                                                            
{'CiphertextBlob': b'', 'KeyId': 'key_id', 'ResponseMetadata': {'HTTPStatusCode': 200, 'HTTPHeaders': {'server': 'amazon.com'}, 'RetryAttempts': 0}}
>>> 
```

```python
>>> import boto3
>>> client_kms = boto3.client('kms', region_name='ap-northeast-1')
>>> client_kms.encrypt(KeyId='<my-kms-id>', Plaintext='')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/yuecen/myworks/dosomething/venv/lib/python3.7/site-packages/botocore/client.py", line 357, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/Users/yuecen/myworks/dosomething/venv/lib/python3.7/site-packages/botocore/client.py", line 661, in _make_api_call
    raise error_class(parsed_response, operation_name)
botocore.exceptions.ClientError: An error occurred (ValidationException) when calling the Encrypt operation: 1 validation error detected: Value at 'plaintext' failed to satisfy constraint: Member must have length greater than or equal to 1
```

moto is an awesome library!  Thank you.

yuecen

**Comment by spulec** [comment]
Thanks; going to set this as a feature request for us to add the validation here.

**Comment by mattsb42-aws** [comment]
Looks like I did not fix this in #2397 :(

I would re-target this to: All KMS APIs need to correctly validate input values.

Key IDs are correctly validated now per #2420, and some of the other values are too, but not all.

I'll try to grab some time to look at this issue this week.

@spulec Feel free to @ me on any KMS-related issues. :)

**Comment by spulec** [comment]
@mattsb42-aws awesome, appreciate it!

Given your role, I'm happy to give you commit access or anything else that would be helpful. Let me know.
